carddrawer/src/main/java/com/meli/android/carddrawer/model/CardDrawerView.java

```diff
@@ -298,20 +308,53 @@ private void show(@NonNull final GenericPaymentMethod genericPaymentMethod) {
         showTag(genericPaymentMethod, genericTagText, genericFrontLayout);
 
         genericPaymentMethod.setPaymentMethodImage(paymentMethodImage);
+
         genericTitle.setText(genericPaymentMethod.getTitle().getText());
         genericTitle.setTextColor(genericPaymentMethod.getTitle().getColor());
-        showGenericPaymentSubtitle(genericPaymentMethod.getSubtitle());
-        frontBackground.setColorFilter(genericPaymentMethod.getBackgroundColor(), PorterDuff.Mode.SRC_ATOP);
-        backBackground.setColorFilter(genericPaymentMethod.getBackgroundColor(), PorterDuff.Mode.SRC_ATOP);
+        genericTitle.setTypeface(genericTitle.getTypeface(), CardDrawerFont.from(genericPaymentMethod.getTitle().getWeight()).getStyle());
+
+        showGenericText(genericPaymentMethod);
+
+        applyBackground(frontBackground, backBackground, genericPaymentMethod);
     }
 
-    protected void showGenericPaymentSubtitle(@Nullable final GenericPaymentMethod.Text subtitle) {
+    private void applyBackground(@NotNull final AppCompatImageView frontBackground,
+                                 @NotNull final AppCompatImageView backBackground,
+                                 @NotNull final GenericPaymentMethod genericPaymentMethod
+    ) {
+        final List<String> gradientColors = genericPaymentMethod.getGradientColors();
+        final int backgroundColor = genericPaymentMethod.getBackgroundColor();
```

Comment:
minor comment, as we use this only on else branch we could initialize this variable there